samples/wifi/ble_coex/README.rst

```diff
@@ -80,34 +78,34 @@ Additional configuration
 
 To enable different test modes, set up the following configuration parameters in the :file:`prj.conf` file:
 
-* Antenna configuration: Use the :kconfig:option:`CONFIG_COEX_SEP_ANTENNAS` Kconfig option to select the antenna configuration.
-  Set it to ``y`` to enable separate antennas and ``n`` to enable shared antenna.
 * Test modes: Use the following Kconfig options to select the required test case:
 
-  * :kconfig:option:`CONFIG_TEST_TYPE_WLAN_ONLY` for Wi-Fi only test
-  * :kconfig:option:`CONFIG_TEST_TYPE_BLE_ONLY` for Bluetooth LE only test
+  * :kconfig:option:`CONFIG_TEST_TYPE_WLAN_ONLY` for Wi-Fi-only test.
+  * :kconfig:option:`CONFIG_TEST_TYPE_BLE_ONLY` for Bluetooth LE-only test.
   * :kconfig:option:`CONFIG_TEST_TYPE_WLAN_BLE` for concurrent Wi-Fi and Bluetooth LE test.
 
   Based on the required test, set only one of these to ``y``.
 * Test duration: Use the :kconfig:option:`CONFIG_WIFI_TEST_DURATION` Kconfig option to set the duration of the Wi-Fi test and :kconfig:option:`CONFIG_BLE_TEST_DURATION` for the Bluetooth LE test.
   The units are in milliseconds.
-  For example, to set the tests for 20 seconds, set the respective values to ``20000``.
+  For example, to set the test for 20 seconds, set these values to ``20000``.
   For the concurrent Wi-Fi and Bluetooth LE test, make sure that both are set to the same duration to ensure maximum overlap.
-* Bluetooth LE configuration: Set the Bluetooth LE connection interval limits using the :kconfig:option:`CONFIG_INTERVAL_MIN` and :kconfig:option:`CONFIG_INTERVAL_MAX` Kconfig options.
+* Bluetooth LE configuration: Set the Bluetooth LE connection interval limits using the :kconfig:option:`CONFIG_INTERVAL_MIN` and :kconfig:option:`CONFIG_INTERVAL_MAX`.
   The units are 1.25 milliseconds.
   For example, ``CONFIG_INTERVAL_MIN=80`` corresponds to an interval of 100 ms (80 x 1.25).
-* Wi-Fi connection: Use the :kconfig:option:`CONFIG_NET_CONFIG_PEER_IPV4_ADDR` Kconfig option to establish a connection to a peer host.
 
-You must configure the following Wi-Fi credentials in the :file:`prj.conf` file:
+* Wi-Fi connection: Configure the following Wi-Fi credentials in the :file:`prj.conf`: appropriately as per the credentials of the access point used for this testing:
+
 
 .. include:: /includes/wifi_credentials_static.txt
 
 .. note::
-   You can also use ``menuconfig`` to configure ``Wi-Fi credentials``.
+   ``menuconfig`` can also be used to configure ``Wi-Fi credentials``.
```

Comment:
Rephrasing is done to maintain consistency, as ‘you’ is not used in the rest of the document.

Reply:
Ok but I think it would be good if we can maintain the consistency with other samples as well. For ex: https://developer.nordicsemi.com/nRF_Connect_SDK/doc-legacy/latest/nrf/samples/wifi/raw_tx_packet/README.html#wi-fi-static-credential-options